src/_locales/el/messages.json

```diff
@@ -376,7 +376,7 @@
         "description": "'About' section text"
     },
     "aboutExtensionDesc": {
-        "message": "Web Scrobbler is a browser extension created for people who listen to music online through their browser, and would like to keep an updated playback history using scrobbling services, such as Last.fm, Libre.fm and ListenBrainz.",
+        "message": "Web Scrobbler is a browser extension created for people who listen to music online through their browser, and would like to keep an updated playback history using scrobbling services, such as Last.fm, Libre.fm, ListenBrainz, and Pleroma.",
```

Comment:
Not sure how this will work? We pull translations from remote, only maintaining source language (en).

Reply:
I just removed all these because I believe I made them in error. Currently the only one that's edited is the English one, which I hope is right.